### ISSUE TITLE
Frequent crashes on start (or shortly after)

### ISSUE THREAD

**Issue by jose1711** [mborik/GPMD85Emulator#25]
Testing the latest master (d1be955) on my system I am getting frequent crashes (SIGABRT). Couple backtraces (not sure if they show anything useful):
```
..
..
DBG: [GUI] Status bar icons resource loaded
DBG: [Screen] Windowed mode: 624x580 -> viewport: 576x512
[New Thread 0x7fffe9c006c0 (LWP 112172)]
DBG: [Sound] Initialized device to 44100Hz/8bit with 1024B (2048B) buffer
corrupted double-linked list

Thread 1 "GPMD85emu" received signal SIGABRT, Aborted.
0x00007ffff787be44 in ?? () from /usr/lib/libc.so.6
(gdb) bt
#0  0x00007ffff787be44 in ?? () from /usr/lib/libc.so.6
#1  0x00007ffff7823a30 in raise () from /usr/lib/libc.so.6
#2  0x00007ffff780b4c3 in abort () from /usr/lib/libc.so.6
#3  0x00007ffff780c354 in ?? () from /usr/lib/libc.so.6
#4  0x00007ffff7886085 in ?? () from /usr/lib/libc.so.6
#5  0x00007ffff7886bfc in ?? () from /usr/lib/libc.so.6
#6  0x00007ffff7889a9a in ?? () from /usr/lib/libc.so.6
#7  0x00007ffff788a6f4 in malloc () from /usr/lib/libc.so.6
#8  0x00007ffff7aae6e5 in operator new (sz=40) at /usr/src/debug/gcc/gcc/libstdc++-v3/libsupc++/new_op.cc:50
#9  0x000055555556c3b2 in ?? ()
#10 0x000055555556c57c in ?? ()
#11 0x0000555555569066 in ?? ()
#12 0x000055555556b055 in ?? ()
#13 0x0000555555560df8 in ?? ()
#14 0x00007ffff780cc88 in ?? () from /usr/lib/libc.so.6
#15 0x00007ffff780cd4c in __libc_start_main () from /usr/lib/libc.so.6
#16 0x0000555555561285 in ?? ()
```

```
..
..
DBG: Starting main CPU 50Hz loop

Thread 1 "GPMD85emu" received signal SIGSEGV, Segmentation fault.
0x00005555555700e6 in ?? ()
(gdb) bt
#0  0x00005555555700e6 in ?? ()
#1  0x0000555555567e74 in ?? ()
#2  0x0000555555561123 in ?? ()
#3  0x00007ffff780cc88 in ?? () from /usr/lib/libc.so.6
#4  0x00007ffff780cd4c in __libc_start_main () from /usr/lib/libc.so.6
#5  0x0000555555561285 in ?? ()
```

I did a manual bisect and found that the last working build was ba4f705. b0fd63b was broken and 266e230 started to crash.

Tested on Arch Linux x86_64, Xorg, removed old emulator config.

**Comment by mborik** [comment]
Already fixed by @a8jan in #24 waiting for the merge, so just be patient. 😉

**Comment by jose1711** [comment]
Thank you, suggested fix confirmed working.
